schema/model_policies.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$udmi_version": "1.5.2",
+  "title": "Policies Model",
+  "description": "Device policies",
+  "type": "object",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "additionalProperties": false,
+  "properties": {
+    "named_policies": {
+      "description": "An array of policies which are applicable to the device",
+      "type": "array",
+      "items": {
+        "type": "string",
+        "pattern": "^[a-z_0-9]*$"
```

Comment:
can you prefix this with a `[a-z]` since I want to reserve a leading `_` for future use, and they shouldn't start with numbers!